### ISSUE TITLE
[Bug] 训练loss NaN

### ISSUE THREAD

**Issue by zbzbzbb95** [open-mmlab/mmpretrain#1811]
### 分支

main 分支 (mmpretrain 版本)

### 描述该错误

您好！
我在使用MAE预训练时，在不同mask_ratio下都出现了loss变为nan的情况。
不同mask_ratio下，在200~300epoch时（loss还在持续下降）忽然出现了nan
我希望可以在loss backward时通过torch.autograd帮助定位问题
with torch.autograd.detect_anomaly():
      loss.backward()
但我并不知道mmpretrain中，loss实际进行反向传播在哪里进行（感觉在optimizer相关的hook中，但始终没有找到），希望给一些指引（具体的文件位置），谢谢！

### 环境信息

none

### 其他信息

none

**Comment by waizuichougou** [comment]
同样出现了这个问题 不知道该怎么解决
